rune/libcontainer/container_linux.go

```diff
@@ -426,6 +426,19 @@ func (c *linuxContainer) deleteExecFifo() {
 	os.Remove(fifoName)
 }
 
+func (c *linuxContainer) includeRuneletFd(cmd *exec.Cmd) error {
+	const selfName = "/proc/self/exe"
+	runeletFd, err := unix.Open(selfName, unix.O_RDONLY|unix.O_CLOEXEC, 0)
+	if err != nil {
+		return err
+	}
+
+	cmd.ExtraFiles = append(cmd.ExtraFiles, os.NewFile(uintptr(runeletFd), selfName))
+	cmd.Env = append(cmd.Env,
+	fmt.Sprintf("_LIBCONTAINER_RUNELETFD=%d", stdioFdCount+len(cmd.ExtraFiles)-1))
+	return nil
+}
+
```

Comment:
Move this chunk into commandTemplate() where a common place contains extra fd passthrough.